arcade/coin/src/main.leo

```diff
@@ -0,0 +1,26 @@
+program puzzle_arcade_coin_v002.aleo {
+  record PuzzleArcadeCoin {
+    owner: address,
+    amount: u64
```

Comment:
do we want to make these u128's following mtsp? https://github.com/demox-labs/aleo-standard-programs/blob/0b508e9f97b4e4b7abf2c1193b4d338f773822b4/multi_token_support_program/src/main.leo#L7